src/main/java/server/Router.java

```diff
@@ -0,0 +1,19 @@
+package main.java.server;
+
+import main.java.server.request.Request;
+
+import java.util.HashMap;
+
+public class Router {
+
+    public String route(Request request) {
+        HashMap<String, String> requestHeaders = request.getHeaders();
+        String path = requestHeaders.get("path");
```

Comment:
This line is making me think that perhaps the `requestHeaders` should be encapsulated in an object. Two reasons:
1. If you want to rename the key (`"path"`), that change will need to leak out of the place where the path is being set on the request headers and into this namespace.
2. If you want to change the implementation of the request headers (e.g. from `HashMap<String, String>` to `ArrayList`), that change will similarly leak out.